launch_env.sh

```diff
@@ -15,3 +15,5 @@ if [ -z "$PASSIVE" ]; then
 fi
 
 export STAGING_ROOT="/data/safe_staging"
+export SKIP_FW_QUERY=1
```

Comment:
Is there something wrong with the FW query for you car?

Reply:
@sshane  Yea nothing was showing up in the carFw section for me. I was able to use the query_fw_versions.py script but had to force the bus to use 500k, as the default script didn't work. This was when I first got my comma, and haven't messed with attempting to get the fw since then.

Reply:
I'd try the fw_versions.py script again, or just do `./fw_versions.py --brand hyundai` to only query with requests for Hyundai

Reply:
@sshane  It gets stuck waiting for the VIN. I let my last attempt go for about a minute then cancelled it. I tried restarting the car during the query, starting the car up when i click enter, and already have the car started. 
comma@tici:/data/openpilot/selfdrive/car$ ./fw_versions.py --brand hyundai --debug
Getting vin...
ISO-TP: REQUEST - 0x7df 0x22f190
ISO-TP: TX - single frame - 0x7df
CAN-TX: 0x7df - 0x0322f19000000000
ISO-TP: REQUEST - 0x18db33f1 0x22f190
ISO-TP: TX - single frame - 0x18db33f1
CAN-TX: 0x18db33f1 - 0x0322f19000000000
^CSIGINT received, exiting

Reply:
Do you have your comma power plugged in for this?

Reply:
Yea have comma power plugged in and car was fully on. I’ll look at the boardd output tomorrow and try to diagnose it more

Reply:
@sshane 
comma@tici:/data/openpilot$ ./selfdrive/boardd/boardd
selfdrive/boardd/main.cc: starting boardd
selfdrive/boardd/boardd.cc: attempting to connect
selfdrive/boardd/boardd.cc: connected to board
selfdrive/boardd/panda.cc: CAN: MALFORMED USB RECV PACKET

I am able to run query_fw_versions with the red panda connected to my computer, but not when it's connected to the c3. I did notice line 468 in uds.py was getting asserted and query_fw_versions was crashing. Once  I handled that, I received the following fw values:
[query_fw_output.txt](https://github.com/commaai/openpilot/files/9103379/query_fw_output.txt)

Reply:
FW issue likely a setup issue, not sw related. Will look into this when I have time to mess with my setup